lib/sql_dust/utils/path_utils.ex

```diff
@@ -125,17 +128,17 @@ require IEx
 
     paths = Regex.scan(regex, sql)
     sql = Regex.replace(regex, sql, fn(match) ->
-      "{#{match}}"
+      "{" <> match <> "}"
     end)
 
     Enum.reduce(paths, {sql, options}, fn([path], {sql, options}) ->
       {path_alias, options} = prepend_path_alias(path, options, true)
-      {String.replace(sql, "{#{path}}", path_alias), options}
+      {String.replace(sql, "{" <> path <> "}", path_alias), options}
     end)
   end
 
   def prepend_path_alias(path, options, cascade \\ false) do
-    {path, column} = dissect_path(path, options)
+    {path, column} = do_dissect_path(path, options)
```

Comment:
I split it up into 2 functions, to not perform list |>  join(".")  |> split(".") as it was previously